commands/TenorCommand.ts

```diff
@@ -56,8 +58,8 @@ export class TenorCommand implements ISlashCommand {
                 },
                 author: {
                     icon: 'https://raw.githubusercontent.com/wreiske/Rocket.Chat.App-Tenor/master/images/Tenor-256.png',
-                    name: `/tenor ${trigger.trim()}`,
-                    link: `https://tenor.com/search/${trigger.trim()}`,
+                    name: (showLink ? `/tenor ${trigger.trim()} ${showLink}` : ''),
+                    link: (showLink ? `https://tenor.com/search/${trigger.trim()}` : ''),
```

Comment:
Does this work okay just making `link` an empty string? Or should we remove `link` completely from the object if showLink is false?

Reply:
As far as I can tell Rocket.Chat will just ignore empty strings. So this should be fine. I just thought it was nicer to read like this. Since link and name are optionals I guess you could do ` : undefinged` instead of ` : ''` if you want to. But it seems to work just fine like this.